src/config/main.ts

```diff
@@ -186,7 +186,7 @@ const configGetter = convict({
   mantis: mantisConfig({
     packageDirectory: path.resolve(defaultDistPackagesDir, 'mantis'),
     executableName: 'mantis',
-    dataDirName: 'mantis',
+    dataDir: null,
```

Comment:
does `null` mean `use default`?

Reply:
> - `dataDir` became `walletDataDir` to better distinguish it from the Mantis datadir config option
> - `mantis.dataDirName` became `mantis.dataDir` to not confuse users (it's a path now)
>   If it's set, it will override the setting option on the UI and the field will be disabled.